### ISSUE TITLE
custom tflite  GPUdelegate build failed on Android P(reno)

### ISSUE THREAD

**Issue by sunzhe09** [keras-team/keras#12813]
<em>Please make sure that this is a Bug or a Feature Request and provide all applicable information asked by the template.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.</em>  

**System information**  
- Have I written custom code (as opposed to using example directory):  
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  
- TensorFlow backend (yes / no):  yes
- TensorFlow version:  1.12.2
- Keras version:  
- Python version:  3.5
- CUDA/cuDNN version:  
- GPU model and memory:  

You can obtain the TensorFlow version with:  
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"  
You can obtain the Keras version with:  
python -c 'import keras as k; print(k.__version__)'  

**Describe the current behavior**  

build GPUdelegate  failed   on android P

**Describe the expected behavior**  
build pass 

**Code to reproduce the issue**  

I convert my model like this:
freeze_graph --input_graph=eval.pb  --input_checkpoint=./model_quant_self/model.ckpt-19-19    --output_graph=frozen_eval_graph.pb  --output_node_names=Softmax  
tflite_convert  --output_file=poolnet_gzq.tflite --graph_def_file=./model_gzq.pb  --inference_type=FLOAT    --input_arrays=Placeholder --input_shapes=1,224,224,3 --output_arrays=oup
  

**Other info / logs**  
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.  

here is the log:

2019-05-13 16:25:04.870 20456-22983/? W/System.err: java.lang.RuntimeException: java.lang.IllegalArgumentException: Internal error: Failed to apply delegate: GpuDelegate Prepare: fuse_auto_input failedNode number 133 (GpuDelegate) failed to prepare.

2019-05-13 16:25:04.870 20456-22983/? W/System.err: Caused by: java.lang.IllegalArgumentException: Internal error: Failed to apply delegate: GpuDelegate Prepare: fuse_auto_input failedNode number 133 (GpuDelegate) failed to prepare.
failedNode133 is my outputNode Sigmiod/Softmax  ,I had tried ,both are failed
